ansible_ai_connect_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -263,6 +272,15 @@ export const useChatbot = () => {
           ...timeoutMessage(),
         };
         addMessage(newBotMessage);
+      } else if (isTooManyRequestsError(e)) {
+        // Insert a 3-sec delay before showing the "Too Many Request" message
```

Comment:
hmmm UI freeze is actually a concern to me...

Reply:
This should not "freeze" UI.  Please watch the recording I posted to Slack.

This delay is a "nice-to-have" and is not necessary for rate limiting, but I think it can give users an impression that the server is busy than just immediately responding with the "Server is busy" message.